theme/redbox.js

```diff
@@ -0,0 +1,31 @@
+// Create a new div element
+var newDiv = document.createElement("div");
+// Set the id attribute of the new div
+newDiv.id = "aspect-ratio-helper";
+// Create a nested div inside the new div
+var nestedDiv = document.createElement("div");
+// Append the nested div to the new div
+newDiv.appendChild(nestedDiv, newDiv.firstChild);
+// Get the parent element where you want to append the new div
+var parentElement = document.body; // Change this to your desired parent element
+// Append the new div to the parent element
+parentElement.insertBefore(newDiv, parentElement.firstChild);
+// Create the button element
+var hideShowButton = document.createElement("button");
+hideShowButton.innerHTML = "redbox";
+hideShowButton.className = "icon-button";
+hideShowButton.type = "button";
+hideShowButton.title = "showing/hiding red box";
```

Comment:
This is shown when hovering over the button, so let's explain a bit about what the red box does:
```suggestion
hideShowButton.title = "Outline the area that fits on one screen while teaching the course.";
```

Reply:
Done